Tests/SonarScanner.MSBuild.PreProcessor.Tests/SonarWebServiceTest.cs

```diff
@@ -99,6 +99,28 @@ public void LogWSOnError()
             }
         }
 
+        [TestMethod]
+        public void WarnIfDeprecated_SQ79_ShouldNotWarn()
+        {
+            this.ws = new SonarWebService(this.downloader, "http://myhost:222", this.logger);
+            this.downloader.Pages["http://myhost:222/api/server/version"] = "7.9.0.2232";
+
+            this.ws.WarnIfSonarQubeVersionIsDeprecated().Wait();
+
+            this.logger.AssertWarningNotLogged("The version of SonarQube you are using is deprecated. Analyses will fail starting 6.0 release of the Scanner for .NET");
```

Comment:
This doesn't scale and will not provide meaningful the result if the message is modified somehow. Let's test empty instead (directly or through some new assertion method on `TestLogger`).
```
this.logger.Warnings.Should().BeEmpty();
```